models/Color/Color.js

```diff
@@ -45,9 +45,30 @@ class Color {
    */
   constructor(args, json) {
     if (json) {
-      Object.assign(this, json);
+      if (json._class === 'swatch') {
```

Comment:
What is this conditional for? Generally what we try to do is have 1 JS class per "_class" in the Sketch JSON, having this conditional in the constructor kinda breaks that. Looking at the raw Sketch JSON, I don't think this would get used.

Reply:
I think the intention was to support the kind of workflow that’s given in the example in #141:

Create a swatch:
```
const redSwatch = new Swatch({
    name: 'Primary/Red',
    color: new Color('#c00')
});
sketch.addSwatch(redSwatch);
```

Then use it in place of a `Color` when creating an object that references the swatch:
```
artboard.addLayer(new Rectangle({
    ...,
    name: 'Red Rectangle',
    style: {
        fills: [
            {
                color: redSwatch, // <- Has _class === 'swatch'
            },
        ],
    },
}));
```

An alternative to supporting swatches anywhere you can use a color would be to add `swatchID` as an optional property of `Color`s and require users pass a `Color` and not a `Swatch` when building other objects.

For example, a `fromColor()` static method that returns a `Color` with the `swatchID` set:
```
artboard.addLayer(new Rectangle({
    ...,
    name: 'Red Rectangle',
    style: {
        fills: [
            {
                color: Color.fromSwatch(redSwatch)
            },
        ],
    },
}));
```

You could also go the other way with a method on `Swatch` that returns a color:
```
artboard.addLayer(new Rectangle({
    ...,
    name: 'Red Rectangle',
    style: {
        fills: [
            {
                color: redSwatch.asColor()
            },
        ],
    },
}));
```

Reply:
That makes sense, thanks! I like the instance method on a Swatch class that returns a color, I think that makes the most sense. In general I think I want to closely map the classes to the classes in the Sketch file format, which there is a "swatch" class and a "color" class, and the "swatch" class has an attribute which is a color. But also the point of this library is to make it easier to interact with instead of just building JSON objects by hand, and your instance method does that. 

If you would like, could you create a new PR with that change in addition to these changes? Or I can do that in a new PR (but I want you to get credit for the contribution).

Reply:
@dbanksdesign Yes I will make a new PR branching off of this.